elements/bulbs-reading-list/lib/reading-list-articles.js

```diff
@@ -35,6 +35,7 @@ export default class ReadingListArticles {
 
   registerEvents () {
     this.dispatcher.on('reading-list-item-clicked', this.handleListItemClicked.bind(this));
+    this.dispatcher.on('reading-list-item-url-changed', this.setCurrentArticle.bind(this));
```

Comment:
just a convention I usually do, but you can set binds in the constructor of  rather than in the same line as calling them.

e.g. `this.setCurrentArticle = this.setCurrentArticle.bind(this)`, same for others. but just /2cents, not blocker